.github/workflows/tests.yml

```diff
@@ -43,9 +43,12 @@ jobs:
         pip install nox
         nox --version
 
-    - name: Run nox
+    - name: Run unit tests
       run: nox -s test-${{ matrix.python }}
 
+    - name: Run integration tests
+      run: nox -s integration-${{ matrix.python }}
```

Comment:
Wait, why do we need a new environment? Can't we just use the same, but have the ``--run-integration`` flag?